airbyte-integrations/connectors/source-shopify/source_shopify/source.py

```diff
@@ -116,7 +116,22 @@ class Orders(IncrementalShopifyStream):
     data_field = "orders"
 
     def path(self, **kwargs) -> str:
-        return f"{self.data_field}.json?status=any"
+        return f"{self.data_field}.json"
+
+    def request_params(self, stream_state=None, next_page_token: Mapping[str, Any] = None, **kwargs) -> MutableMapping[str, Any]:
+        stream_state = stream_state or {}
+        if next_page_token:
+            params = {"limit": self.limit, **next_page_token}
+        else:
+            params = {
+                "limit": self.limit,
+                "order": f"{self.primary_key} asc",
+                "created_at": self.start_date,
```

Comment:
is this `created_at_min`?

Reply:
It is. Corrected, it appears after the latest experiments and tests, so just missed to change back.